<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_sys_manage_operatorinformation_Dialog</class>
 <widget class="QDialog" name="w_sys_manage_operatorinformation_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
     <item>
      <widget class="lds_tableView" name="tableView"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>手工优惠限额</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_num_present">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_vch_operid"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>送菜金额</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>送菜控制</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_int_discount">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>操作密码</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">color:blue;</string>
         </property>
         <property name="text">
          <string>注:新增操作员的操作密码同操作代码</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>折扣底限(%)</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="lds_ComboBox" name="comboBox_ch_pre_mode"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>操作代码</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="lds_CheckBox" name="checkBox_ch_flag">
         <property name="text">
          <string>是否生效</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_num_free">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>操作姓名</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_vch_password"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_vch_operator"/>
       </item>
       <item row="4" column="0">
        <widget class="lds_CheckBox" name="checkBox_ch_waiter">
         <property name="text">
          <string>是否服务员</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="lds_CheckBox" name="checkBox_ch_free_flag">
         <property name="text">
          <string>免单控制</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="lds_toolbar" name="widget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar</class>
   <extends>QFrame</extends>
   <header>lds_toolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>lds_tableView</class>
   <extends>QTableView</extends>
   <header>lds_tableview.h</header>
  </customwidget>
  <customwidget>
   <class>lds_ComboBox</class>
   <extends>QComboBox</extends>
   <header>lds_combobox.h</header>
  </customwidget>
  <customwidget>
   <class>lds_CheckBox</class>
   <extends>QCheckBox</extends>
   <header>lds_checkbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableView</tabstop>
  <tabstop>lineEdit_vch_operid</tabstop>
  <tabstop>lineEdit_vch_operator</tabstop>
  <tabstop>lineEdit_vch_password</tabstop>
  <tabstop>checkBox_ch_flag</tabstop>
  <tabstop>checkBox_ch_free_flag</tabstop>
  <tabstop>checkBox_ch_waiter</tabstop>
  <tabstop>lineEdit_num_free</tabstop>
  <tabstop>lineEdit_num_present</tabstop>
  <tabstop>lineEdit_int_discount</tabstop>
  <tabstop>comboBox_ch_pre_mode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
